common/protob/messages.proto

```
@@ -406,6 +407,7 @@ enum MessageType {
     MessageType_AptosSignedTx = 10603 [(wire_out) = true];
     MessageType_AptosSignMessage = 10604 [(wire_in) = true];
     MessageType_AptosMessageSignature = 10605 [(wire_out) = true];
+    MessageType_AptosSignSIWAMessage = 10606 [(wire_in) = true];
```

Buf (1.55.1)

Enum value name "MessageType_AptosSignSIWAMessage" should be UPPER_SNAKE_CASE, such as "MESSAGE_TYPE_APTOS_SIGN_SIWA_MESSAGE".

(ENUM_VALUE_UPPER_SNAKE_CASE)